src/firefly/js/core/MasterSaga.js

```diff
@@ -30,6 +31,7 @@ export function* masterSaga() {
     // This section starts any predefined Sagas
     yield fork( watchCatalogs, null, dispatch, getState);
     yield fork( imagePlotter, null, dispatch, getState);
+    yield fork( syncCharts, null, dispatch, getState);
```

Comment:
This is for predefined sagas,  we may want to move the list to ReduxFlux where all the other list of things that are initialized are. 

>> where are we suppose to initialize these sagas? What if it's not needed?

I don't understand the questions.

We current have  many more sagas than these three they are more short lived and dynamically started in certain context using `dispatchAddSaga`.